packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -415,3 +415,20 @@ class RenderingFlutterBinding extends BindingBase with GestureBinding, ServicesB
     renderView.child = root;
   }
 }
+
+/// Client for listening to system fonts change.
+///
+/// Using this mixin will cause it automatically subscribe to system channel.
+mixin SystemFontsClient on RenderObject {
```

Comment:
Maybe call this "RelayoutWhenSystemFontsChangeMixin" to better describe what it does?